src/util/LazyJsonParser.cpp

```diff
@@ -250,7 +252,7 @@ std::optional<nlohmann::json> LazyJsonParser::constructResultFromParsedChunk(
   size_t nextChunkStart =
       materializeEnd == 0 ? 0 : std::min(materializeEnd + 1, input_.size());
   if (input_.size() - nextChunkStart >= 1'000'000) {
-    throw std::runtime_error("Ill formed Json.");
+    throw Error("Ill formed JSON.");
```

Comment:
This message should be something like
`QLever currently doesn't support SERVICE results where a single result row is larger than 1 MB`.